src/explore-education-statistics-admin/src/pages/release/footnotes/components/FootnoteForm.tsx

```diff
@@ -35,10 +35,25 @@ const FootnoteForm = ({
 }: Props) => {
   const initialValues = useMemo<BaseFootnote>(() => {
     const subjects = mapValues(footnoteMeta.subjects, subject => {
-      const { indicators, filters } = subject;
+      const { indicators, filters, subjectId } = subject;
+
+      let subjectSelectionType = 'NA';
+
+      if (footnote && Object.keys(footnote.subjects).includes(subjectId)) {
+        const foundSubject = footnote.subjects[subjectId];
+        if (foundSubject.selected) {
+          subjectSelectionType = 'All';
+        } else if (
+          !foundSubject.selected &&
+          (Object.keys(foundSubject.filters).length > 0 ||
```

Comment:
Don't think we need to do `> 0` as any non-zero value here would be truthy.  Just nit-picking though!